Project/src/MakeCall/MakeCall.js

```diff
@@ -778,6 +789,10 @@ this.deviceManager.on('selectedSpeakerChanged', () => { console.log(this.deviceM
                             <div className="ms-Grid-col ms-lg6 ms-sm6 mb-4">
                                 <h2>Placing and receiving calls</h2>
                                 <div>{`Permissions audio: ${this.state.permissions.audio} video: ${this.state.permissions.video}`}</div>
+                                <MediaConstraint
+                                    onChange={this.handleMediaConstraint}
+                                    disabled={this.state.call || !this.state.loggedIn}
+                                />

```

Comment:
can we move it down below all inputs to start the call?

Reply:
Moved the UI down to the bottom of `Placing and receiving calls`